### PR TITLE
Revamp "add sighting" modal

### PR DESCRIPTION
**Pull request by norrisng-bc**
* Completely overhaul "add sighting" modal to only ask for: 
  * Date range
  * (Sub)region
  * Moose count
* Sightings page has been updated to reflect
* Cleanup
  * Remove geolocation code
  * Cleanup other unused code and saga actions

*Note:* tick hair loss as an additional field has not been implemented yet, but it will be displayed in the sightings page once so.

### CODE REVIEW

File: app/src/UI/Map.tsx [hunk 1]
```diff
@@ -174,18 +174,17 @@ const MapMarkers = (props: any) => {
...
-        const position = getOffsetLocation(index, markerPosition);
-        const mooseIcon = getMooseIcon(moose, "Current");
-        return (
-          <Marker key={index} position={position} icon={mooseIcon} />
-        );
-      })}
-    </>
-  );
+  // const Meese = (props: any) => (
+  //   <>
```
Comment (brettedw) [inline app/src/UI/Map.tsx:178]: delete too?

**Review by brettedw — APPROVED** [review]
👌

File: app/src/UI/Map.tsx (hunk 1 above)
Comment (norrisng-bc) [reply app/src/UI/Map.tsx:178]: Leaving it in for now, as we will need to display moose icons later (albeit not at an exact lat/lon position)

File: app/src/UI/Map.tsx [hunk 2]
```diff
@@ -197,33 +196,33 @@ const MapMarkers = (props: any) => {
...
-            </Marker>
-          )
-        })
-      })}
-    </>
-  )};
+  //     return(
+  //   <>
+  //     {allSightings?.map((sighting: any, index: number) => {
+  //       return sighting.mooseArray.map((moose: any, mooseIndex: number) => {
```
[the remark this answers is not in the document]
Comment (norrisng-bc) [reply app/src/UI/Map.tsx:204]: Leaving it in for now, as we will need to display moose icons later (albeit not at an exact lat/lon position)

**Pull request merged**